app/src/Controller/AppController.php

```diff
@@ -59,5 +60,6 @@ public function beforeFilter(\Cake\Event\EventInterface $event)
         // for all controllers in our application, make index and view
         // actions public, skipping the authentication check
         $this->Authentication->addUnauthenticatedActions(['index', 'view']);
+        $this->Authorization->skipAuthorization();
```

Comment:
Is it OK to call the skipAuthorization for all controllers (extending this AppController) & actions (beforeFiler called before action executed)?

Reply:
Please refer the CakePHP tutorial https://book.cakephp.org/4/en/tutorials-and-examples/cms/authorization.html

> Lastly we’ll mark the add, login, and logout actions as not requiring authorization by adding the following to src/Controller/UsersController.php

> The skipAuthorization() method should be called in any controller action that should be accessible to all users even those who have not logged in yet.

> Lastly add the following to the tags, view, and index methods on the ArticlesController

Reply:
I'm sorry, https://github.com/ArielGalvez/tutorial-cakephp/pull/5#discussion_r1377549692 had caused confusion. 🙇 

I checked again when I reviewed Mauricio PR and beforeFilter is the best place to set the skipAuthorization. Please kindly refer https://github.com/combimauri/tutorial-cakephp/pull/5#discussion_r1384831850

Reply:
thanks! I was trying alternative solutions for that 😄